packages/host/app/components/operator-mode/container.gts

```diff
@@ -602,14 +653,24 @@ export default class OperatorModeContainer extends Component<Signature> {
       <CardCatalogModal />
 
       <div class='operator-mode__with-chat {{this.chatVisibilityClass}}'>
-        <SubmodeSwitcher
-          @submode={{this.operatorModeStateService.state.submode}}
-          @onSubmodeSelect={{this.updateSubmode}}
-          class='submode-switcher'
-        />
+        <div class='operator-mode__top-menu'>
+          <SubmodeSwitcher
+            @submode={{this.operatorModeStateService.state.submode}}
+            @onSubmodeSelect={{this.updateSubmode}}
+          />
+          {{#if this.isCodeMode}}
+            <CardURLBar
+              @url={{this.codePath}}
+              @onEnterPressed={{this.loadCardForCodeMode}}
+              @card={{this.cardForCardMode}}
+              @notFoundError={{this.cardNotFoundError}}
```

Comment:
i would rename this to `codeModeCardError` that makes it more clear what this error is about.